roles/server_create_delete/tasks/main.yml

```diff
@@ -12,6 +12,8 @@
       key_name: "{{ server_keypair_name }}"
       network: "{{ server_net }}"
       security_groups: "{{ security_group }}"
+      # WARN: template and use of vars may cause replacement of \" to \'
+      userdata: "{{ lookup('template', 'coreos-ignition.json.j2') | to_json | string }}"
```

Comment:
so you are adding linux user with this userdata right?

Reply:
Yeah, I was trying to make this configurable, but use of lookup template and messing with variables resulted multiple times in the userdata containing \' instead of \", what coreos rejects. ATM passing ignition to i.e. standard Fedora is working ok. Maybe we should have a separate "server" module invocation with userdata for coreos mode, but I didn't want to have it now